### PR TITLE
Refactor Service arguments

### PR DESCRIPTION
**Pull request by chriskuehl**
Based on feedback from @bukzor in another review.

### CODE REVIEW

File: pgctl/service.py [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
-        self.path = path
-        self._pgctl_app = pgctl_app
+class Service(namedtuple('Service', ['path', 'scratch_dir'])):
 
     def __repr__(self):
-        return "Service('{path}')".format(path=self.path.strpath)
+        return "Service(path='{path}', scratch_dir='{scratch_dir}')".format(
+            path=self.path.strpath,
+            scratch_dir=self.scratch_dir.strpath,
+        )
```
Comment (bukzor) [inline pgctl/service.py]: I believe this is the default repr for a namedtuple.

Comment (chriskuehl) [inline pgctl/service.py]: true except that `self.path` and `self.scratch_dir` are py.path objects

so the repr would be:
Service(path=local(u'/tmp/magic-service'), scratch_dir=local(u'/tmp/magic-service-scratch'))

if you're ok with that (I am) i can drop the __repr__

Comment (bukzor) [inline pgctl/service.py]: yep. 

less code, less bugs

Comment (chriskuehl) [inline pgctl/service.py]: removed

**Pull request merged**